optuna/trial/_frozen.py

```diff
@@ -113,12 +113,18 @@ def objective(trial):
             Datetime where the :class:`~optuna.trial.Trial` finished.
         params:
             Dictionary that contains suggested parameters.
+        distributions:
+            Dictionary that contains parameter distributions.
         user_attrs:
             Dictionary that contains the attributes of the :class:`~optuna.trial.Trial` set with
             :func:`optuna.trial.Trial.set_user_attr`.
+        system_attrs:
+            Dictionary that contains the attributes of the :class:`~optuna.trial.Trial` set with
+            :func:`optuna.trial.Trial.set_system_attr`.
         intermediate_values:
             Intermediate objective values set with :func:`optuna.trial.Trial.report`.
-
+        trial_id:
+            ID of the :class:`~optuna.trial.Trial`
```

Comment:
FrozenTrial provides `_trial_id` attribute which is not public to users. So I'd say we don't need to list `trial_id` here. What do you think?

Reply:
Thank you for your input!
There is no `trial_id` attribute actually. It should not be listed in `Attributes` section since it's a constructor argument.

We may be able to document `trial_id` argument in `Args` section. However, I cannot find such an example that document `Args` section in docstring for Python classes. Do you think we should add `Args` section in this docstring?
https://google.github.io/styleguide/pyguide.html#384-classes

Reply:
Sorry, I misunderstood. I agree that it should not be listed in `Attributes` section.